File: Lab01/C_Lang/src/Demo04.cpp
//
//  Created by NDHuy on 04/05/2020
//  Copyright 2020 by SEstudio
//  Input one integer number, which is the number of year.
//  Check this year is Leap year?
//

#include <stdio.h>

int main()
{
    int year;
    printf("Input a number of year: ");
    scanf("%d", &year);

    bool flagLeapYear = ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0);

    if (flagLeapYear)    
        printf("%d is leap year\n", year);
    else 
        printf("%d is NOT leap year\n", year);

    return 0;
}

File: Lab01/Python_Lang/Demo04.py
#
#  Created by NDHuy on 04/05/2020
#  Copyright 2020 by SEstudio
#  Input one integer number, which is the number of year.
#  Check this year is Leap year?
#

def main():
    year = int(input("Input a number of year: "))

    flagLeapYear = ((year % 4 == 0) and (year % 100 != 0)) or (year % 400 == 0)

    if flagLeapYear:
        print(str(year) + " is leap year")
    else:
        print(str(year) + " is NOT leap year")


if __name__ == "__main__":
    main()